### PR TITLE
Update scale-app documentation

### PR DESCRIPTION
**Pull request by MartinJJones**
Updating following the 2nd line drill - https://trello.com/c/fPEXeNU4/2019-2nd-line-drill-scaling-up-an-application

### CODE REVIEW

File: source/manage-app/scale-app/index.html.md [hunk 1]
```diff
@@ -11,8 +11,9 @@ If user demand on your app increases, you must increase the resources available
 
 ## Scaling your app
 
-1. Change your app's `values.yaml` file in the [`charts` folder in the `govuk-helm-charts` repo](https://github.com/alphagov/govuk-helm-charts/blob/main/charts/).
-1. Commit this change.
+1. Determine if you need to update your app's `values.yaml` or enviroment yaml file, for example - [`values-integration.yaml`](https://github.com/alphagov/govuk-helm-charts/blob/main/charts/app-config/values-integration.yaml)
```
Comment (sengi) [inline source/manage-app/scale-app/index.html.md:14]: This new step poses a question without answering it, which is not a helpful thing in a playbook.

I'm not clear on what the problem was that you're trying to solve by adding this step. Can you tell me more about what you were trying to do?

`app-config/values-<env>.yaml` should normally be the correct place to make these changes in this use case.

Comment (MartinJJones) [reply source/manage-app/scale-app/index.html.md:14]: Thanks for the feedback Chris, fair points.

We wanted to update this following a drill on 2nd line, as we wasn't sure where to make the change initially. The documentation currently links to the `charts` directory, but we could not find the `values.yml` file to change. I think we were able to find where to make the change in the end by searching the repo and looking at previous pull requests.

We wanted to update the documentation and include an example link to one of the value.yml to change.

Comment (sengi) [reply source/manage-app/scale-app/index.html.md:14]: Aha thanks, that's helpful. So yeah for the purposes of this playbook, the right place to make the change is `values-<environment>.yaml` for the environment where you want to scale up/down.

So I think the playbook was just linking to the wrong place and that understandably caused some confusion.

Comment (sengi) [reply source/manage-app/scale-app/index.html.md:14]:
```suggestion
1. Find the Helm values file for the environment that you want to change, for example [`values-integration.yaml`](https://github.com/alphagov/govuk-helm-charts/blob/main/charts/app-config/values-integration.yaml).
```

Comment (MartinJJones) [reply source/manage-app/scale-app/index.html.md]: Thanks for the suggestion 👍

Comment (sengi) [reply source/manage-app/scale-app/index.html.md]: no probs, thanks for finding the error and fixing it!

**Review by ChrisBAshton — APPROVED** [review]

**Pull request merged**